src/openapi/streaming/serializer/serializer-json.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * JSON Serialization
+ * @constructor
+ */
+function SerializerJson(name) {

```

Comment:
Consider creating interface for both Serialization methods we now support. However, JSON at this point is just push through, as singlalR already parses to JSON.